src/Model/Translatable/TranslatableMethodsTrait.php

```diff
@@ -130,7 +131,7 @@ public static function getTranslationEntityClass(): string
      *
      * @param string $locale The locale (en, ru, fr) | null If null, will try with current locale
      */
-    protected function doTranslate(?string $locale = null, bool $fallbackToDefault = true): TranslationInterface
+    protected function doTranslate(?string $locale = null, bool $fallbackToDefault = true)
```

Comment:
Not a big fan, but it prevents the following issue:

![Capture d’écran de 2020-02-04 07-18-40](https://user-images.githubusercontent.com/2103975/73719115-a91c0c00-471e-11ea-9b71-2cf2578860ba.png)

Also this is a protected method which is only consumed by `translate()` (which still have `TranslationInterface` php return type)

Reply:
Don't let tools force you to write bad code just to make them happy.
This destroys contract completely, must be reverted.

Reply:
`doTranslate()` is a protected method which is not part of `TranslatableInterface` interface, so the contract is not broken. 
The method is only used by public method `translate()` which still have `TranslatableInterface` return type, so I think it's okay?